CHANGES.rst

```diff
@@ -23,6 +23,8 @@ New Features
   - Change behavior to warn about units that are not FITS-compliant when
     writing a FITS file but not when reading. [#5675]
 
+- ``astropy.io.fits.diff``
+
```

Comment:
You need to add an actual change log entry under existing FITS bullet point. See other entries for example.